components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -165,7 +165,9 @@ tweaking their $case/namelist_scream.xml file.
     <!-- P3 microphysics -->
     <p3 inherit="physics_proc_base">
       <do_prescribed_ccn>true</do_prescribed_ccn>
+      <do_prescribed_ccn COMPSET=".*SCREAM*noSPA*">false</do_prescribed_ccn>
       <do_predict_nc>true</do_predict_nc>
+      <do_predict_nc COMPSET=".*SCREAM*noSPA*">false</do_predict_nc>
```

Comment:
Similarly, change to 
<do_predict_nc COMPSET=".*SCREAM.*noSPA">false</do_predict_nc>